OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -752,8 +765,8 @@ public void onClick(View v) {
             environment.getAnalyticsRegistry().trackScreenView(
                     Analytics.Screens.SECTION_OUTLINE, courseData.getCourseId(), courseComponent.getInternalName());
         }
-
         fetchLastAccessed();
+        detectDeepLinking();
```

Comment:
Please move `courseComponentId = courseComponent.getId();` from `line:717` to `line:763` after calling `detectDeepLinking()`

Reply:
done.